ports/geckolib/gecko_bindings/tools/regen_style_structs.sh

```diff
@@ -102,8 +102,10 @@ export RUST_BACKTRACE=1
   -match "pair"                                                     \
   -match "SheetParsingMode.h"                                       \
   -match "StaticPtr.h"                                              \
+  -match "nsProxyRelease.h"                                         \
```

Comment:
Opaque types would work, but we'd need those opaque types to look like:

```rust
pub enum nsMainThreadHolder<T> {}
pub enum nsMainThreadHandle<T> {}
```

but currently using `-opaque-type` I get:

```rust
pub enum nsMainThreadHolder {}
pub enum nsMainThreadHandle {}
```

which doesn't work with the types in bindings.rs we generate:

```rust
pub type ThreadSafePrincipalHolder = nsMainThreadPtrHolder<nsIPrincipal>;
pub type ThreadSafeURIHolder = nsMainThreadPtrHolder<nsIURI>;
```